src/p_settings.hxx

```diff
@@ -822,16 +822,19 @@ private:
 
 class AspectRatioSetting final : public Settings::IntSetting {
 public:
-    enum Kind { FULL, WIDE, ULTRAWIDE };
+    enum Kind { FULLOPENMATTE, FULL, WIDE, ULTRAWIDE };
```

Comment:
In order to add a new aspect ratio, you should also add a new case to the switch found in `BetterSMS::getScreenRenderWidth()` and `BetterSMS::getScreenOrthoWidth()`.